models/zeroex/celo/zeroex_celo_api_fills.sql

```diff
@@ -0,0 +1,264 @@
+{{  config(
+    tags=['dunesql'],
+        alias = alias('api_fills_celo'),
+        materialized='incremental',
+        partition_by = ['block_date'],
```

Comment:
We can use `block_month` here as well.
```suggestion
        partition_by = ['block_month'],
```